files/zh-tw/web/http/status/304/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 304 Not Modified
+slug: Web/HTTP/Status/304
+l10n:
+  sourceCommit: fb8c438501d119fb897a9667ee85c1486f2c03d6
+---
+
+{{HTTPSidebar}}
+
+HTTP **`304 Not Modified`** 重新導向回應碼表示無需重新傳送所請求的資源。
+當請求是帶有 {{HTTPHeader("If-None-Match")}} 或 {{HTTPHeader("If-Modified-Since")}} 標頭的條件式 {{HTTPMethod("GET")}} 或 {{HTTPMethod("HEAD")}} 請求，並且該條件求值為 false 時，會發送此回應碼。如果條件求值為 true，則這是對一個已緩存的資源的隱式重定向，而該資源的請求如果沒有條件，則會導致一個 {{HTTPStatus("200")}} `OK` 的回應。
+
+回應不得包含主體，並且必須包括與等效的 {{HTTPStatus("200")}} `OK` 回應中會發送的標頭：
+{{HTTPHeader("Cache-Control")}}、{{HTTPHeader("Content-Location")}}、{{HTTPHeader("Date")}}、{{HTTPHeader("ETag")}}、{{HTTPHeader("Expires")}} 和 {{HTTPHeader("Vary")}}。
+
+> **備註：** 許多[開發者工具的網絡面板](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/index.html)會創建多餘的請求，導致出現 `304` 的回應，這樣開發人員就可以查看對本地緩存的訪問，所以建議在此情況下使用 `304` 狀態碼。
+
+## 狀態
+
+```http
+304 Not Modified
+```
+
+## 範例
+
+下面的範例顯示了使用 curl 進行的帶有條件請求標頭的 {{HTTPMethod("GET")}} 請求，以及收到的 HTTP 回應。如果我們知道自 2050 年 11 月 21 日以來資源已被更新，則第一個範例將返回一個 {{HTTPStatus("200")}} `OK`。為了說明，請求使用了未來日期 2050 年 11 月 21 日，以檢查自該日期以來是否已更新資源：
```

Comment:
```suggestion
下面的範例顯示了使用 curl 進行的帶有條件請求標頭的 {{HTTPMethod("GET")}} 請求，以及收到的 HTTP 回應。如果我們知道自 {{HTTPHeader("If-Modified-Since")}} 標頭指定的時間戳以來資源已被更新過，則第一個範例將返回 {{HTTPStatus("200")}} `OK`。為了說明，請求使用了未來日期 2050 年 11 月 21 日，以檢查自該日期以來是否已更新資源：
```